lib/Optimizer/Dialect/FIROps.cpp

```diff
@@ -1042,6 +1042,20 @@ static mlir::ParseResult parseWhereOp(OpAsmParser &parser,
   return mlir::success();
 }
 
+mlir::LogicalResult
```

Comment:
Is there a mechanism to add a unit test for fir::WhereOp;:fold in case the structure of the Fir changes someday?

Reply:
I'll make a note to see what MLIR did.